### DISCUSSION TITLE
Simple Demo Landing Page - Capsule Corp from Dragon Ball franchise

### DISCUSSION THREAD

**Discussion by AlexTechNoir** [vercel/next.js#37261] [Show and tell]
Simple Demo Landing Page made on Next.js - Capsule Corporation from Dragon Ball franchise

In a Readme you can find the links to code examples of some animations I've made

Demo: https://capsule-corp.netlify.app/
GitHub: https://github.com/AlexTechNoir/capsule-corp

**Comment by xyntechx** [comment]
YES Dragon Ball 🔥!!!
